xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/XWikiGroupSheet.xml

```diff
@@ -210,14 +215,20 @@
       return;
     }
     addMembersButton.prop('disabled', true);
-    $.post(form.attr('action'), parameters).done(function() {
+    $.post(form.attr('action'), parameters).done(function(response) {
       // Clear the list of selected users and groups.
       $('#userInput')[0].selectize.clear(true);
       $('#groupInput')[0].selectize.clear(true);
       // Reload the live table.
       editgrouptable.refresh();
       // Show notification message.
-      new XWiki.widgets.Notification("$services.localization.render('xe.admin.groups.addSuccess')", 'done', {timeout: 5});
+      if(response.ignored.size() &gt; 0) {
+        var concat = response.ignored.join(', ');
+        new XWiki.widgets.Notification("$services.localization.render('xe.admin.groups.addIgnored') (" + concat + ")",
```

Comment:
sounds better to create a translation with a parameter than using a concat here. Besides "concat" is not really a good variable name.

Reply:
+1 for parameters, you are never really sure if the placement you have in English is going to work for all locales.

Reply:
I agree, the problem is that IIUC the velocity code is evaluated at the generation of the script, but the content to used as the translation parameter is only available at runtime.

Reply:
The translation is now done server side (see https://github.com/xwiki/xwiki-platform/pull/1374/files#diff-675b8cf1e4313bb08807f805a03daa40R50)